Makefile

```diff
@@ -145,22 +145,20 @@ verify-golangci:
 
 .PHONY: verify-govulncheck
 verify-govulncheck:
-	@if ! command -v govulncheck &>/dev/null; then \
-		go install -mod=mod golang.org/x/vuln/cmd/govulncheck@latest ; \
-	fi
-	govulncheck ./...
+	./scripts/fetch_tools.sh govulncheck && \
+	./_output/bin/govulncheck ./...
 
 # We use the IGNORE var instead of `shellcheck disable=all` for files that are taken from
 # upstream and we wish to keep them with out added shellcheck declaratives.
 .PHONY: verify-sh
 verify-sh:
-	IGNORE="configure-ovs*.sh" ; \
+	IGNORE="configure-ovs.sh" ; \
 	./scripts/fetch_tools.sh shellcheck && \
```

Comment:
Maybe we can eventually have a wrapper script for this step like we do for verify-py.sh